README.md

```diff
@@ -26,12 +26,18 @@ sudo apt-get install git python-dev python-pip
 sudo pip install virtualenv virtualenvwrapper --ignore-installed six
 ```
 
-# Quick setup
+# Setup
 This should be run from your home directory:
 ```
 source <(curl -s https://raw.githubusercontent.com/dimagi/commcare-cloud/master/control/init.sh)
 ```
 
+Alternately:
+```
+git clone https://github.com/dimagi/commcare-cloud.git
+source commcare-cloud/control/init.sh
+```
+
 You may now use `commcare-cloud` or its shorter alias `cchq` whenever you're in the virtualenv.
```

Comment:
I'm still a proponent of suggesting people add

```
export PATH=$PATH:~/.commcare-cloud/bin
source ~/.commcare-cloud/repo/control/.bash_completion
```

to their profile, at least until we've figured out a real way to install this. Super convenient to be able to run `commcare-cloud`/`cchq` form anywhere outside the env.

Reply:
updated

Reply:
sweet, thanks!